src/SoCreate.Extensions.Logging/ActivityLogger/ActivityLogger.cs

```diff
@@ -19,33 +19,42 @@ public ActivityLogger(ILoggerProvider loggerProvider, IConfiguration configurati
         {
             _logger = ((LoggerProvider)loggerProvider).Logger;
             _activityLogType = configuration.GetValue<string>("ActivityLogger:ActivityLogType") ?? "DefaultType";
-            _version = configuration.GetValue<string>("ActivityLogger:ActivityLogVersion") ?? "v1";
+            _version = configuration.GetValue<string>("ActivityLogger:ActivityLogVersion") ?? "1.0.0";
         }
 
         public void LogActivity<TActivityEnum>(
-            IActivityKeySet keySet, 
-            TActivityEnum actionType,
+            int key, 
+            TActivityEnum keyType,
+            int? accountId, 
+            int tenantId,
             AdditionalData? additionalData, 
             string message, 
             params object[] messageData)
         {
-            if (actionType == null)
+            if (keyType == null)
             {
-                throw new ArgumentNullException(nameof(actionType), "actionType must be set");
+                throw new ArgumentNullException(nameof(keyType), "keyType must be set");
             }
             var properties = new List<ILogEventEnricher>
             {
                 new PropertyEnricher(Constants.SourceContextPropertyName, typeof(TSourceContext)),
                 new PropertyEnricher("Version", _version),
-                new PropertyEnricher("KeySet", keySet.ToDictionary()),
-                new PropertyEnricher("ActionType", actionType.ToString(), true),
-                new PropertyEnricher(CosmosActivityLoggerLogConfigurationAdapter.LogTypeKey, _activityLogType)
+                new PropertyEnricher("Key", key.ToString()),
+                new PropertyEnricher("KeyType", keyType.ToString(), true),
+                new PropertyEnricher("TenantId", tenantId),
+                new PropertyEnricher(SqlServerLoggerLogConfigurationAdapter.LogTypeKey, _activityLogType)
```

Comment:
Isn't new PropertyEnricher("AcccountId", accountId), supposed to be listed here?